classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student) unless @students.include?(student)
+    stident.classroom = self
```

Comment:
- [ ] Kindly fix your typo error here. It should be `student`, not `stident`

Reply:
Done. Thanks for the review